src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/PersonPersister.kt

```diff
@@ -15,50 +15,48 @@ class PersonPersister(
   /**
    * Returns the total number of successfully persisted people.
    */
-  fun persistPeople(people: List<Person>): Int {
+  fun persistPeople(people: Sequence<Person>): PersistenceResult {
```

Comment:
Now that we are using sequences we don't know in advance how many rows there are to be processed hence we need to send back how many were persisted and how many failed to give us a idea of how many rows were actually processed.